views/pricing.html

```diff
@@ -57,8 +57,8 @@ <h1 class="text-center">Membership Plans</h1>
         </tr>
         <tr class="signup pricing">
           <td></td>
-          <td><a href="/login" class="btn btn-warning">Sign in</a></td>
-          <td><a href="mailto:support@datahub.io?subject=Becoming a Premium Member of the DataHub" class="btn btn-warning">Sign up</td>
+          <td><a href="/login" class="btn btn-warning" onclick="trackOutboundLink('sign-in')">Sign in</a></td>
+          <td><a href="mailto:support@datahub.io?subject=Becoming a Premium Member of the DataHub" class="btn btn-warning" onclick="trackOutboundLink('sign-up')">Sign up</td>
           <td><a href="mailto:support@datahub.io?subject=Interested in DataHub On Demand Pricing">Contact us</a></td>
```

Comment:
can we add similar tracking on two "Contat us" buttons as `contact-us-pricing-page`